src/generators/Silk.NET.SilkTouch.Scraper/ClangScraper.cs

```diff
@@ -57,6 +59,51 @@ public IEnumerable<Symbol> ScrapeXML(XmlDocument document)
         return visitor.Visit(bindings);
     }
 
+    private static string GetXCodeSDKPath()
+    {
+        var process = new Process();
+        process.StartInfo = new ProcessStartInfo("xcrun", "--show-sdk-path")
+        {
+            RedirectStandardOutput = true
+        };
+        process.Start();
+        process.WaitForExit();
+        return process.StandardOutput.ReadToEnd();
+    }
+    
+    /// <summary>
+    /// Resolves platform specific standard includes
+    /// </summary>
+    /// <returns>An enumerable of directories to include</returns>
+    /// <remarks>
+    /// On Windows, this attempts to use the Visual Studio Resolver
+    /// On OSX, this attempts to run `xcrun --show-sdk-path` to get the SDK path, to then include /sdk path>/usr/include
+    /// On other UNIX platforms, this simply defaults to /usr/include
+    /// </remarks>
+    public IEnumerable<string> ResolveStandardIncludes()
+    {
+        if (OperatingSystem.IsWindows())
+        {
+            if (VisualStudioResolver.TryGetVisualStudioInfo(out var info))
+            {
+                if (info.UcrtIncludes.Length > 0)
+                {
+                    foreach (var include in info.UcrtIncludes)
+                    {
+                        yield return include;
+                    }
+                }
+                else
```

Comment:
Don't we want both?

Reply:
I think this is the ucrt vs msvc code? GH lost track of the snippet.
Not an expert, just quickly googled online, and there I got the impression ucrt is just a newer rewrite of C(++) standard headers, while msvc is the older stuff?

Reply:
I think the naming is misleading, the ucrt array is likely to only contain windows sdk headers and the MSVC tools contains the standard C headers.

Reply:
Ah, I see. Will change in that case :+1: